### PR TITLE
feat(stylelint): update documentation to tell consumers that they can…

### PR DESCRIPTION
**Pull request by ThomasRumasLM**
… custom stylelint's rules

## I have read [the contributing guidelines](https://mozaic.adeo.cloud/Contributing/)

- [X] Yes
- [ ] No

## Does this PR introduce a breaking change?

read [What is a breaking change ?](https://mozaic.adeo.cloud/Contributing/Prerequisite/GitConventions/#breaking-changes-)

- [ ] Yes
- [X] No

## Describe the changes

Update stylelint's documentation to tell consumers that they can custom the rules

Github issue number or Jira issue URL: N/A

### CODE REVIEW

File: src/docs/GetStarted/Developers/configureStylelint/index.mdx [hunk 1]
```diff
@@ -61,6 +61,25 @@ Please checkout [Mozaic configuration](/GetStarted/Developers/Configuration/#sty
 
 </Highlight>
 
+### Add your own stylelint rules
```
Comment (tiloyi) [inline src/docs/GetStarted/Developers/configureStylelint/index.mdx]: ~~Add your own stylelint rules~~ => Add custom rules

File: src/docs/GetStarted/Developers/configureStylelint/index.mdx [hunk 2]
```diff
@@ -61,6 +61,25 @@ Please checkout [Mozaic configuration](/GetStarted/Developers/Configuration/#sty
 
 </Highlight>
 
+### Add your own stylelint rules 
+
+It can appear that sometimes you need to use custom rules in stylelint configuration file for your project, you can add it this way in `stylelint.config.js`:
```
Comment (tiloyi) [inline src/docs/GetStarted/Developers/configureStylelint/index.mdx]: Sometimes you may need to customize Stylelint rules or add new ones.

To do this, you can expand your `stylelint.config.js` file this way:

File: src/docs/GetStarted/Developers/configureStylelint/index.mdx [hunk 3]
```diff
@@ -61,6 +61,25 @@ Please checkout [Mozaic configuration](/GetStarted/Developers/Configuration/#sty
...
+  'color-no-invalid-hex': null,
+  'font-family-no-duplicate-names': null
+};
+
+const stylelintConfig = mozaicRules;
+stylelintConfig.rules = {...mozaicRules.rules, ...customRules};
+
+module.exports = stylelintConfig;
+```
+
```
Comment (tiloyi) [inline src/docs/GetStarted/Developers/configureStylelint/index.mdx]: I think you should also add a "warning" highlight to this section (just below the title), to indicate that this tip is for advanced users only. Something like this: 

```MDX
<Highlight theme="warning" title="Only for advanced users">
The trick below is for advanced users only.
By doing this evolution you go out of the rules set and planned by Mozaic. 
As a result, we cannot fully guarantee the optimal functioning of Mozaic. So be sure of what you are doing.
</Highlight>
````

File: src/docs/GetStarted/Developers/configureStylelint/index.mdx (hunk 2 above)
Comment (nherwegh) [inline src/docs/GetStarted/Developers/configureStylelint/index.mdx]: It appears (…)

**Pull request merged**